.github/workflows/cherry-picks-warn.yml

```diff
@@ -21,5 +21,17 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: '### 🚨 Important Reminder:\nAre you cherry-picking commits to a release branch? If so, please ensure you use the `Rebase and merge` option when merging this pull request. Do not use the `Squash and merge` option, as it makes reverting individual commits impossible. If the `Rebase and merge` option is disabled (it usually is), follow [Configuring commit rebasing for pull requests](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-rebasing-for-pull-requests) to temporarily enable it, and disable it once the PR is merged.'
+              body: `# ⚠️ Important: Cherry-Pick Merge Instructions
```

Comment:
I believe this message is still valuable when `Squash and merge` is somehow enabled.